core/ir/x86/decode_table.c

```diff
@@ -7118,7 +7117,8 @@ const instr_info_t evex_prefix_extensions[][2] = {
  */
 const instr_info_t rex_b_extensions[][2] = {
   { /* rex.b extension 0 */
-    {OP_nop,  0x900000, "nop", xx, xx, xx, xx, xx, no, x, tpe[103][2]},
+    /* we chain these even though encoding won't find them */
+    {OP_nop,  0x900000, "nop", xx, xx, xx, xx, xx, no, x, tpe[103][3]},
     /* For decoding we avoid needing new operand types by only getting
      * here if rex.b is set.  For encode, we would need either to take
```

Comment:
So this comment is explaining why the encoder won't find this?  Wondering if we should file a separate issue so we can point at it in the test and PR description even if we don't plan to fix it.

Reply:
Yes, and done.